docs/dev/table/sqlClient.md

```diff
@@ -151,6 +151,17 @@ Mode "embedded" submits Flink jobs from the local machine.
                                            multiple times.
      -s,--session <session identifier>     The identifier for a session.
                                            'default' is the default identifier.
+     -u,--update <SQL update statement>    Experimental (for testing only!):
```

Comment:
Actually not documenting this was on purpose.

Reply:
Thanks, @twalthr , I see.. It is experimental and for test only..revert it.